### DISCUSSION TITLE
Examples of Apps in production

### DISCUSSION THREAD

**Discussion by metarama** [aws-samples/aws-saas-factory-ref-solution-serverless-saas#42] [Q&A] [answered]
Please list any apps in production you know for sure that use this ref app template or architecture.

**Comment by anubhavynr** [comment, marked as answer]
The intention of this solution is to provide a reference point/sample implementation and not intended to be used in production as is. It is licensed using MIT-0. Having said that, many customers have used this reference solution to bootstrap their SaaS architecture by leveraging the code provided. I cannot provide you the list of customers unfortunately.
